charts/argo-workflows/templates/controller/workflow-sa.yaml

```diff
@@ -21,5 +21,20 @@ metadata:
 imagePullSecrets:
     {{- toYaml . | nindent 2 }}
   {{- end }}
+  {{- if $.Values.workflow.serviceAccount.createSecret }}
+secrets:
+  - name: {{ $.Values.workflow.serviceAccount.name }}.service-account-token
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/service-account-token
+metadata:
+  name: {{ $.Values.workflow.serviceAccount.name }}.service-account-token
+    {{- with $namespace }}
+  namespace: {{ . }}
+    {{- end }}
+  annotations:
+    kubernetes.io/service-account.name: "{{ $.Values.workflow.serviceAccount.name }}"
+  {{- end }}
```

Comment:
Can we also add additional annoations ( `.Values.workflow.serviceAccount.annotations` ) and labels ( `.Values.workflow.serviceAccount.labels` ) ? 🤔 
e.g.) https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/argocd-application-controller/serviceaccount.yaml#L8-L18